lib/archive_importer/post_importer.rb

```diff
@@ -17,6 +17,7 @@ def substitute_author
 
       entity_data["photos"].each do |photo|
         photo["entity_data"]["author"] = user.diaspora_handle
+        photo["entity_data"]["remote_photo_path"] = "#{AppConfig.pod_uri}uploads\/images\/"
```

Comment:
Okay, so this is how we link uploaded photos. If there are files in the photos archive that weren't in the end linked to any photo instance, should we detect and clean them up?